tests/rest/client/v2_alpha/test_capabilities.py

```diff
@@ -148,3 +170,17 @@ def test_get_does_include_msc3244_fields_when_enabled(self):
             self.assertGreater(len(details["support"]), 0)
             for room_version in details["support"]:
                 self.assertTrue(room_version in KNOWN_ROOM_VERSIONS, str(room_version))
+
+    def _test_capability(self, capability: str, access_token: str, expect_success=True):
+        """
+        Requests the capabilities from server and check if the value is expected.
+        """
+        channel = self.make_request("GET", self.url, access_token=access_token)
+        capabilities = channel.json_body["capabilities"]
+
+        self.assertEqual(channel.code, 200)
+
+        if expect_success:
+            self.assertTrue(capabilities[capability]["enabled"])
+        else:
+            self.assertFalse(capabilities[capability]["enabled"])
```

Comment:
```suggestion
        self.assertEqual(capabilities[capability]["enabled"], expect_success)
```

... though to be honest, I would be inclined to have it return `capabilities` and leave the `self.assertTrue(capabilities["m.change_password"]["enabled"])` or similar in the caller.